jbpm-test-coverage/src/test/java/org/jbpm/test/functional/timer/GlobalQuartzDBTimerServiceTest.java

```diff
@@ -305,9 +305,6 @@ public void afterNodeLeft(ProcessNodeLeftEvent event) {
         
         countDownListener.waitTillCompleted(3000);
         assertEquals(2, timerExporations.size());
-
-        
-        manager.close();
```

Comment:
This is a unit test which I wrote in the past. I removed the line because I noticed that manager is closed by GlobalTimerServiceBaseTest.cleanup().